build.gradle

```diff
@@ -64,6 +64,7 @@ dependencies {
     implementation('io.prometheus:simpleclient') {
         because('metrics to export')
     }
+    implementation group: 'org.apache.commons', name: 'commons-collections4', version: '4.4'
```

Comment:
You use this library only for creating a HashSet. You can use `Set.of` from java to do that

Reply:
oh indeed